.drone/drone.yml

```diff
@@ -1410,6 +1410,11 @@ services:
   - name: cgroup
     path: /sys/fs/cgroup
 steps:
+- commands:
+  - apk add --no-cache bash git
+  - git fetch origin --tags
+  image: alpine
+  name: fetch-tags
```

Comment:
Adding `git fetch` to this pipeline assures that we do not revert the change that introduced `git fetch` in the Makefile [here](https://github.com/grafana/loki/pull/8232).